evm/src/trace/identifier/signatures.rs

```diff
@@ -29,7 +29,7 @@ pub struct SignaturesIdentifier {
 }
 
 impl SignaturesIdentifier {
-    #[tracing::instrument(name = "signaturescache")]
+    #[tracing::instrument(target = "forge::signatures")]
```

Comment:
can the tracing instrument target be moved to the impl block so you dont need it in every function?

Reply:
I'm not sure, at least it's not covered in the docs https://docs.rs/tracing/0.1.10/tracing/attr.instrument.html